src/common/util/src/os/lin/lin_shared_object_loader.cpp

```diff
@@ -56,5 +48,18 @@ void* get_symbol(const std::shared_ptr<void>& shared_object, const char* symbol_
     }
     return procAddr;
 }
+
+void SharedObjectCloser::operator()(void* shared_object) const {
+    if (shared_object) {
+        if (0 != dlclose(shared_object)) {
+            std::cerr << "t dlclose failed";
+            if (auto error = dlerror()) {
+                std::cerr << ": " << error;
+            }
+            std::cerr << std::endl;
```

Comment:
The general approach seems to be use `OPENVINO_DEBUG` to print messages in such cases